app/routers/likes.py

```
@@ -0,0 +1,15 @@
+from fastapi import APIRouter, FastAPI, HTTPException
+
+router = APIRouter(prefix="/posts/{post_id}",tags=["likes"])
+
+@router.post("/likes",summary="좋아요 등록",description= "로그인한 사용자가 특정 게시글에 좋아요를 등록하는 리소스.", tags=["likes"])
+async def post_likes(post_id: int):
```

Ruff (0.14.11)

Unused function argument: `post_id`

(ARG001)